### PR TITLE
BZ1957067: Explain the route domain configuration

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.8+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=1957067

Link to docs preview: https://52393--docspreview.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-creating-a-route_route-configuration

https://52393--docspreview.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html#nw-ingress-configuring-application-domain_configuring-ingress

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by lihongan** [comment]
/lgtm
Thank you!

File: modules/nw-creating-a-route.adoc [hunk 1]
```diff
@@ -73,8 +73,8 @@ spec:
     kind: Service
     name: hello-openshift
 ----
-<1> `<Ingress_Domain>` is the default ingress domain name.
-<2> `targetPort` is the target port on pods that is selected by the service that this route points to.  
+<1> `<Ingress_Domain>` is the default ingress domain name. The `ingresses.config/cluster` object gets created during the installation. You cannot change the `ingresses.config/cluster` object. If you want to specify a different domain, you can specify an alternative cluster domain using the `appsDomain` option.
```
Comment (tmalove) [inline modules/nw-creating-a-route.adoc:76]:
```suggestion
<1> `<Ingress_Domain>` is the default ingress domain name. The `ingresses.config/cluster` object is created during the installation and cannot be changed. If you want to specify a different domain, you can specify an alternative cluster domain using the `appsDomain` option.
```

Comment (xenolinux) [reply modules/nw-creating-a-route.adoc]: Addressed

File: networking/routes/route-configuration.adoc [hunk 2]
```diff
@@ -66,3 +66,8 @@ include::modules/nw-ingress-edge-route-default-certificate.adoc[leveloffset=+1]
 include::modules/nw-ingress-reencrypt-route-custom-cert.adoc[leveloffset=+1]
 
 include::modules/nw-router-configuring-dual-stack.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information on specifying an alternative cluster domain, see xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option].
```
Comment (tmalove) [inline networking/routes/route-configuration.adoc:73]:
```suggestion
* For more information about specifying an alternative cluster domain, see xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option].
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage

Comment (xenolinux) [reply networking/routes/route-configuration.adoc]: acked

File: networking/routes/route-configuration.adoc [hunk 3]
```diff
@@ -66,3 +66,8 @@ include::modules/nw-ingress-edge-route-default-certificate.adoc[leveloffset=+1]
 include::modules/nw-ingress-reencrypt-route-custom-cert.adoc[leveloffset=+1]
 
 include::modules/nw-router-configuring-dual-stack.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about specifying an alternative cluster domain, see xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option].
```
Comment (jeana-redhat) [inline networking/routes/route-configuration.adoc:73]: Add'l resources are no longer supposed to have sentences per Jupiter compatibility
```suggestion
* xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option]
```

**Comment by jeana-redhat** [comment]
@xenolinux looks like this will need manual intervention for 4.8-4.10. I'll merge the other two as soon as they build.

**Comment by jeana-redhat** [comment]
I think you are all set!

**Pull request merged**